src/TLuaInterpreter.cpp

```diff
@@ -13769,3 +13770,21 @@ int TLuaInterpreter::startPermSubstringTrigger(const QString & name, const QStri
 
 }
 
+int TLuaInterpreter::alert( lua_State * L )
+{
+    int luaDurationMs;
+    if( ! lua_isnumber( L, 1 ) )
+    {
+        lua_pushstring( L, "alert: wrong argument type" );
```

Comment:
The current "standard" for this type of message indicates the argument number and what is wrong with the supplied one, I think it should be:

    lua_pushfstring(L, "alert: bad argument #1 type (alert duration in milliseconds as number expected, got %s!)",
                    luaL_typename(L, 1));

Reading the documentation (see below) this number could be optional - which would be doable but needs some recoding to support it (`lua_gettop(L)` would be zero if no argument is supplied and that could be used as a conditional around these lines that capture `luaDurationMs` which could be initialized to zero to cover the case when it is omitted).